docs/installation/teams/index.md

```diff
@@ -4,245 +4,31 @@ title: "Microsoft Teams"
 sidebar_position: 5
 ---
 
-:::info
+## Botkube Microsoft Teams App Versions
 
-Botkube is currently working on a beta version of the **new Botkube for Microsoft Teams** bot. If you are a Microsoft Teams user we would love to have you as a part of the beta program. The new bot will feature:
+In order to connect with Microsoft Teams, Botkube requires a Microsoft Teams application. There are two versions of the Botkube Microsoft Teams App available:
 
-- Single-click installation of Botkube via Microsoft AppSource
-- Interactive command builder for simple interaction with Kubernetes for non-experts
-- No need for ingress to your Kubernetes cluster to use Botkube with Microsoft Teams
+### Botkube Cloud Microsoft Teams App
 
-Please join the beta program for the new Botkube for Microsoft Teams bot, help us shape the future direction of Botkube for Teams, and get early access to the new Teams bot.
+The Botkube Cloud Microsoft Teams App offers several advanced features:
 
-Sign up to find out more about the Botkube for Microsoft Teams beta by sending us a message on the [Contact Us](https://botkube.io/contact) page. You can also leave a comment on the Botkube for Microsoft Teams [GitHub Issue](https://github.com/kubeshop/botkube/issues/1212).
+- Simplified installation into your Microsoft Teams workspace
+- Multi-cluster executor support with a single Teams App
+- Manage Slack channels directly from the Botkube web app and ensure the Botkube bot is invited to the correct channels
 
-:::
+The Botkube Cloud Slack App uses Botkube's cloud services to manage channels and route source events and executor commands. Currently, it requires a manual side-loading of the app, but we are working on getting it listed in the Microsoft Teams App Store.
```

Comment:
```suggestion
The Botkube Cloud Teams App uses Botkube's cloud services to manage channels and route source events and executor commands. Currently, it requires a manual side-loading of the app, but we are working on getting it listed in the Microsoft Teams App Store.
```

BTW it would be good to use one naming conventions, right now I saw:
-  Botkube Cloud MS Teams App
-  Botkube Cloud Teams App
-  Botkube Cloud app
-  Botkube Cloud Web app

would be good to unify it across those docs